src/source/batchUploadDocumentsFromFile.ts

```diff
@@ -54,6 +54,7 @@ export class BatchUploadDocumentsFromFiles {
         await create(platformClient, fields);
       }
       await this.consumer.consume(files);
+      await strategy.postUpload();
```

Comment:
in relation to https://github.com/coveo/push-api-client.js/pull/117/files#r838077559,

```suggestion
      await strategy.postUpload?.();
```